packages/by-components/src/charts/bar_chart.rs

```diff
@@ -0,0 +1,254 @@
+#![allow(non_snake_case)]
+
+use crate::{charts::d3, theme::ChartTheme};
+use dioxus::prelude::*;
+use serde_wasm_bindgen::to_value;
+use wasm_bindgen::{prelude::Closure, JsCast, JsValue};
+
+use super::utils::closure;
+
+#[component]
+pub fn BarChart(
+    width: String,
+    height: String,
+    data: Vec<BarChartData>,
+    #[props(extends = GlobalAttributes)] attributes: Vec<Attribute>,
+    children: Element,
+) -> Element {
+    let chart_theme: ChartTheme = try_use_context().unwrap_or_default();
+    let colors = chart_theme.bar_chart_colors;
```

Comment:
```suggestion
    let color: ColorTheme = try_use_context().unwrap_or_default();
    let colors = color.chart.bar_chart_colors;
```
컬러를 전부 각각 context 로 관리하면 사용할때는 편한데, 나중에 컬러를 바꿀때 관리하기가 힘들어 질거 같아서, color theme 을 통해서 사용하는 걸로 리팩토링했습니다. https://github.com/biyard/rust-sdk/pull/58

Reply:
넵 수정했습니다! @hackartists